options.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+add_action( 'admin_init', 'amp_register_settings' );
+add_action( 'admin_menu', 'amp_custom_admin_menu' );
```

Comment:
We need an easy way to disable this the options page in certain contexts (e.g. across all of WordPress.com). To do so, we could:

- Rename `amp_register_settings` to `amp_admin_init`
- Move the `admin_menu` hook into `amp_admin_init`
- Add a filter in `amp_admin_init` that allows skipping registration of the admin pages.